.github/workflows/ci.yml

```
@@ -24,8 +24,11 @@ jobs:
       - name: Lint
         run: npm run lint
 
-      - name: Build
-        run: npm run build
+      - name: Typecheck
```

Is type check two words? 🤷 
```suggestion
      - name: Type check
```

---

It looks like the TypeScript docs show both, bit I think separate words might be a bit more prominent. I'll take you change!